public/index.php

```diff
@@ -0,0 +1,38 @@
+<?php
```

Comment:
This will be big bc break for users, and our infrastructure team. Need to be described in UPGRADE.md